src/commands.rs

```diff
@@ -209,6 +211,13 @@ fn run_server_process() -> Result<ServerStartup> {
         v.push(0);
         v
     };
+    let workdir = exe_path
+        .parent()
+        .expect("executable path has no parent?!")
+        .as_os_str()
+        .encode_wide()
+        .chain(Some(0u16))
```

Comment:
Hmm, I'm surprised that there isn't a better existing abstraction that we can use here that encapsulates these technical details. Something like:
```
let workdir: WindowsStringParameterThinger = exe_path
    .parent()
    .expect(...)
    .as_os_str()
    .into::<WindowsStringParameterThinger>();

// ...

CreateProcessW(
    ...,
    workdir.to_ptr()
    ...,
)
```

Reply:
There is https://crates.io/crates/widestring but we're not currently using it. I just used the same pattern as already in use in the same file.